code/web/src/modules/user/api/actions.js

```diff
@@ -1,120 +1,161 @@
   if (token) {
-    axios.defaults.headers.common['Authorization'] = `Bearer ${token}`;
+    axios.defaults.headers.common["Authorization"] = `Bearer ${token}`;
   } else {
-    delete axios.defaults.headers.common['Authorization'];
+    delete axios.defaults.headers.common["Authorization"];
   }
 
-  return { type: SET_USER, user }
+  return { type: SET_USER, user };
 }
 
 // Login a user using credentials
 export function login(userCredentials, isLoading = true) {
-  return dispatch => {
+  return (dispatch) => {
     dispatch({
       type: LOGIN_REQUEST,
-      isLoading
-    })
-
-    return axios.post(routeApi, query({
-      operation: 'userLogin',
-      variables: userCredentials,
-      fields: ['user {name, email, role}', 'token']
-    }))
-      .then(response => {
-        let error = ''
+      isLoading,
+    });
+
+    return axios
+      .post(
+        routeApi,
+        query({
+          operation: "userLogin",
+          variables: userCredentials,
+          fields: ["user {name, email, role}", "token"],
+        })
+      )
+      .then((response) => {
+        let error = "";
 
         if (response.data.errors && response.data.errors.length > 0) {
-          error = response.data.errors[0].message
-        } else if (response.data.data.userLogin.token !== '') {
-          const token = response.data.data.userLogin.token
-          const user = response.data.data.userLogin.user
+          error = response.data.errors[0].message;
+        } else if (response.data.data.userLogin.token !== "") {
+          const token = response.data.data.userLogin.token;
+          const user = response.data.data.userLogin.user;
 
-          dispatch(setUser(token, user))
+          dispatch(setUser(token, user));
 
-          loginSetUserLocalStorageAndCookie(token, user)
+          loginSetUserLocalStorageAndCookie(token, user);
         }
-
         dispatch({
           type: LOGIN_RESPONSE,
-          error
-        })
+          error,
+        });
       })
-      .catch(error => {
+      .catch((error) => {
         dispatch({
           type: LOGIN_RESPONSE,
-          error: 'Please try again'
+          error: "Please try again",
+        });
+      });
+  };
+}
+
+//set user style based on survey results
+export function setStyle(surveyResults) {
+  let stylePreference = surveyResults;
+  // return dispatch => {
+  //   dispatch({
+  //     type: LOGIN_REQUEST,
+  //     style: null
+  //   })
+  return (dispatch) => {
+    return axios
+      .post(
+        routeApi,
+        mutation({
+          operation: "addStyleToUser",
+          variables: { surveyResults },
+          fields: ["id", "name", "email", "style"],
         })
+      )
+      .then((response) => {
+        if (response.data.data.addStyleToUser.style) {
+          stylePreference = response.data.data.addStyleToUser.style;
+          return dispatch({
+            type: SET_STYLE,
+            stylePreference,
+          });
```

Comment:
Awesome job getting axios working!

Reply:
stylePreference - that makes sense of using just style